x/lockup/client/cli/query.go

```diff
@@ -535,7 +536,48 @@ $ %s query lockup account-locked-longer-duration <address> <duration>
 	return cmd
 }
 
-// GetCmdAccountLockedLongerDurationNotUnlockingOnly returns account locked records with longer duration from unlocking only queue.
+// GetCmdAccountLockedDuration returns account locked records with a specific duration
+func GetCmdAccountLockedDuration() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "account-locked-duration <address> <duration>",
+		Short: "Query account locked records with a specific duration",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query account locked records with a specific duration.
+Example:
+$ %s query lockup account-locked-duration <address> <duration>
```

Comment:
If we're gonna include an example in the `Long` description, we should actually make it a true example. I.e. fill out the `address` and `duration` values 👍

Reply:
Are we wanting the example to work? We would need an address that always had bonded assets for 2 different durations.

Reply:
Gotcha, so the command itself doesn't have to return results that satisfy an example usage's result, we just need an example usage.

Reply:
@alexanderbez If we want to go this route for `Long` descriptions, we should have constants that signify commonly used arguments. Eg: `EXP_BECH_32` `EXP_DURATION` `EXP_POOL`

Reply:
Can we introduce these constants here?

Reply:
@p0mvn Make a new ticket for this as this would impact basically all commands.